middleware/tls/README.md

```diff
@@ -0,0 +1,13 @@
+# tls
```

Comment:
Yes. I think the *root* middleware already has this problem. Last one counts there (I think).